### DISCUSSION TITLE
Cutlet is producing pinyin output

### DISCUSSION THREAD

**Discussion by shhovhan** [polm/cutlet#42] [Q&A] [answered]
Hello,
I tried cutlet in my python project to romanize japanese characters, but it seems that there is a problem with kanji characters. 
Here is the code snippet

```
def romanize_string(name):
    katsu = cutlet.Cutlet()
    return katsu.romaji(name)

name = '通勤経路及び手当支給申請書'
print(romanize_string(name))
```
The result of this code snippet is `Tong Qin Jing Lu Ji biShou Dang Zhi Gei Shen Qing Shu`, which is not correct. Correct romanized text should be something like this: `Tsuukin keiro oyobi teate shikyuu shinseisho`.

Could you please take a look and help me to understand this issue?

Thanks

**Comment by polm** [comment, marked as answer]
Cutlet doesn't know anything about pinyin so it's impossible for it to produce the `Tong Qin` output. Running your code with `import cutlet` at the top produced `Tsuukin keiro oyobi teate shikyuu shinseisho` for me. 

Perhaps you have another function with the same name in your code somewhere?

**Reply by shhovhan** [reply]
Thank you @polm. I had unidecode in my code which was changing the string name before sending it to my function.
